Client/Frontend/Onboarding/OnboardingCardViewController.swift

```diff
@@ -288,6 +328,12 @@ class OnboardingCardViewController: UIViewController, Themeable {
         delegate?.showNextPage(viewModel.cardType)
     }
 
+    @objc
+    func linkButtonAction() {
+        viewModel.sendTelemetryButton(isPrimaryAction: false)
```

Comment:
This will need to change as well. Button telemetry is no longer primary & secodary action, but also with a link action. So we should react accordingly.